features/sendConsentFormEmail.feature

```diff
@@ -1,18 +1,17 @@
 @email
 Feature: Test set for sending emails
+
 	Background:
 		Given user arrives on the Schul-Cloud homepage
 
 	@sendConsentFormEmail
-	Scenario Outline: Send emails to all students without a full declaration of consent.
+	Scenario Outline: As a user, I want to be able to send emails to all students without a full declaration of consent.
 		When <userRole> logs in with email '<adminsUsername>' and password '<adminsPassword>'
 		And <userRole> performs first login actions: data protection acceptance
 		When <userRole> goes to administration
 		And <userRole> goes to students administration
```

Comment:
Probably will result in a merge conflict. On develop a step has been added here. Please have a look.